src/euphorie/content/widgets/logo.py

```diff
@@ -0,0 +1,10 @@
+from plone.formwidget.namedfile.widget import NamedImageWidget
+from z3c.form.widget import FieldWidget
+
+
+class LogoWidget(NamedImageWidget):
+    pass
```

Comment:
You could limit the accepted types via the `accept` attribute, if you want.
e.g.:
```
class LogoWidget(NamedImageWidget):
    accept = ("*.png", "*.jpg", "*.gif",  "*.webp")
```
... or maybe add the accept parameter in the directives statement above.

Note: out of my head, untested....

See: https://github.com/plone/plone.namedfile/pull/158